### ISSUE TITLE
Misleading release branch names

### ISSUE THREAD

**Issue by jandubois** [cloudfoundry-incubator/quarks-operator#1031]
This is really more untidiness than an actual bug: The repo has many release branch names of the form `vN.0.x` which really should be `vN.x` (or maybe even `release-N` because using the `v` naming scheme for both tags and branch names can lead to confusion).

```
$ git branch -r | grep -E /v[0-9]
  origin/v0.4.x
  origin/v1.0.x
  origin/v2.0.x
  origin/v3.0.x
  origin/v4.0.x
  origin/v5.0.x
```

The `v4.0.x` has many tags beyond the ones matching the implied pattern:

```
$ git log --oneline --simplify-by-decoration origin/v4.0.x
ffc6f942 (tag: v4.5.6, origin/v4.0.x) Fix typo
891ad2fe (tag: v4.5.5) Disable test that won't pass anymore on CI
2df377d9 (tag: v4.5.4) Disable test that won't pass anymore on CI
48230888 (tag: v4.5.3) Disable test that won't pass anymore on CI
870c6162 (tag: v4.5.2) fix: remove focus in test
6d16e3c0 (tag: v4.5.1) fix: interpolation for explicit vars that depend on implicit vars
385b1698 (tag: v4.5.0) Merge remote-tracking branch 'origin/master' into v4.0.x
501885ec (tag: v4.4.5) Merge remote-tracking branch 'origin/master' into v4.0.x
b984473a (tag: v4.4.4) Merge remote-tracking branch 'origin/master' into v4.0.x
9bf7466d (tag: v4.4.3) Merge remote-tracking branch 'origin/master' into v4.0.x
d3125cfa (tag: v4.4.2) Merge remote-tracking branch 'origin/master' into v4.0.x
05bae26c (tag: v4.4.1) Merge remote-tracking branch 'origin/master' into v4.0.x
a7028ee1 (tag: v4.4.0) Merge remote-tracking branch 'origin/master' into v4.0.x
157049a1 (tag: v4.3.1) Merge remote-tracking branch 'origin/master' into v4.0.x
d5b4d820 (tag: v4.3.0) Merge remote-tracking branch 'origin/master' into v4.0.x
0a6fec85 (tag: v4.2.0) Merge remote-tracking branch 'origin/master' into v4.0.x
8162d1c5 (tag: v4.1.0) Merge remote-tracking branch 'origin/master' into v4.0.x
52a73efb (tag: v4.0.0) fix: error styling
...
```

**Comment by viovanov** [comment]
👍we'll use `6.x` when version 6 comes out

**Comment by manno** [comment]
v6.x please, not sure the scripts can handle it without the v

**Comment by manno** [comment]
I believe this is fixed with v6.0
